morango/parsers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework.parsers import BaseParser
+import gzip
```

Comment:
Probably best to put this import inside the `parse` method, as importing it at this level could already throw the exception. Since this module isn't specific to gzip, we would want to be able to import from this file without errors.